jekyll/_cci2/server/installation/phase-2-core-services.adoc

```diff
@@ -0,0 +1,806 @@
+[#create-helm-values]
+== 3. Create helm values
+
+Before installing CircleCI, it is recommended to create a new `values.yaml` file unique to your installation. <<example-manifests, Here>> are some example values.yaml files that are a good place to start.  The following describes the minimum required values to include in `values.yaml`. Additional customizations are available, see the provided `values.yaml` for all available options.
+
+For sensitive data there are two options:
+
+* add into the `values.yaml` file
+* add them as Kubernetes secrets directly
+
+This flexibility allows you to manage secrets using whichever process you prefer.
+
+NOTE: Whichever option you choose, this sensitive information is stored as a kubernetes secret within CircleCI.
+
+[#api-token]
+=== a. API token
+
+The application requires a secret containing an API token. This API token is used to facilitate internal API communication to api-service. Use a random string and store it securely, CircleCI will not be able to recover this value if lost. There are two options depending on whether you want to create the secret, or if you want CircleCI to create it for you.
+
+[.tab.apitoken.You_create_secret]
+--
+**Option 1:** Create the secret yourself.
+
+[source,shell]
+----
+kubectl create secret generic api-token \
+  --from-literal=api-token=<your-super-secret-random-value>
+----
+--
+
+[.tab.apitoken.CircleCI_creates_secret]
+--
+**Option 2:** CircleCI creates the secret for you.
+
+Add the value to `values.yaml`. CircleCI will create the secret automatically.
+
+[source,yaml]
+----
+apiToken: <your-super-secret-random-value>
+----
+--
+
+[#session-cookie]
+=== b. Session cookie
+
+The application requires a session cookie key secret, which CircleCI uses to sign session cookies. The key secret must be exactly 16 characters long. Use a random string and store it securely, CircleCI will not be able to recover this value if lost. There are two options depending on whether you want to create the secret, or if you want CircleCI to create it for you.
+
+[.tab.sessioncookie.You_create_secret]
+--
+**Option 1:** Create the secret yourself.
+
+[source,shell]
+----
+kubectl create secret generic session-cookie \
+--from-literal=session-cookie-key=<your-secret-key-16-chars>
+----
+--
+
+[.tab.sessioncookie.CircleCI_creates_secret]
+--
+**Option 2:** CircleCI creates the secret for you.
+
+Add the value to `values.yaml`.  CircleCI will create the secret automatically.
++
+[source,yaml]
+----
+sessionCookieKey: <your-secret-key-16-chars>
+----
+--
+
+[#encryption]
+=== c. Encryption
+
+The application requires a secret containing signing and encrpytion keysets. These keysets are used to encrypt and sign artifacts generated by CircleCI. <<server/installation/phase-1-prerequisites#encryption-signing-keys,These keys were created during the prerequisites phase>>. CircleCI will not be able to recover the values if lost. Depending on how you prefer to manage secrets, there are two options.
```

Comment:
Maybe shorten the link title here: <<server/installation/phase-1-prerequisites#encryption-signing-keys,These keys were created during the prerequisites phase>> as this pushes the whole phrase to a new line. Maybe just "created during the prerequisites phase" as the link title?